app/models/sns.go

```diff
@@ -66,3 +66,17 @@ type SubscriptionAttributes struct {
 	//ReplayPolicy        string                 `json:"ReplayPolicy" schema:"ReplayPolicy"`
 	//ReplayStatus        string                 `json:"ReplayStatus" schema:"ReplayStatus"`
 }
+
+// DeleteTopicV1
+
+func NewDeleteTopicRequest() *DeleteTopicRequest {
+	return &DeleteTopicRequest{}
+}
+
+type DeleteTopicRequest struct {
+	TopicArn string `json:"TopicArn" schema:"TopicArn"`
+}
+
+func (r *DeleteTopicRequest) SetAttributesFromForm(values url.Values) {
+	r.TopicArn = values.Get("TopicArn")
```

Comment:
I don't think this will hurt anything, but I think the `schema` tag will help it find this.  This method pattern is really for the stuff like `Attribute.1.name` > `Attribute.1.value` so we can find each thing and use the 2 parts across both pieces to allocate it properly.

So I might remove this unless you had another need for it?  It should be ok for the interface if the method is just empty.